packages/cadl-rlc-emitter/test/integration/generated/authentication/apiKey/typescript.json

```diff
@@ -8,5 +8,6 @@
     "version": "1.0.0"
   },
   "isCadlTest": true,
-  "title": "AuthApiKeyClient"
+  "title": "AuthApiKeyClient",
+  "generateTest": true
```

Comment:
Can you add a test where only sample generation is enabled. I remember had some issues when build the code with code generated by such configuration.

Reply:
Sure but pls notice we didn't enable the sample generation in cadl.

Reply:
Issue created to track this bug: https://github.com/Azure/autorest.typescript/issues/1590